ranger/config/rifle.conf

```diff
@@ -227,6 +228,7 @@ ext ace, has unace = unace l "$1" | less
 ext ace, has unace = for file in "$@"; do unace e "$file"; done
 ext rar, has unrar = unrar l "$1" | less
 ext rar, has unrar = for file in "$@"; do unrar x "$file"; done
+ext rar|zip, has mcomix,     X, flag f = mcomix -- "$@"
```

Comment:
Let's have this in the Documents section of rifle.conf, I think that makes more sense.

Reply:
you get a lot of comic archives as .rar or .zip
This is not pointless

Reply:
That's not what I said. Please reread my remark carefully, I'm only asking to move the condition into the cb[rz] rule, nothing more. Having it there would make other people test whether their favorite comic readers support `.zip` and `.rar`.

Reply:
it's meant to be low priority, it's not a general archive manager
what you say, will make it come first with rar/zip files.

Reply:
well, it doesn't work with atril, but works with qcomicbook
didn't test zathura